### PR TITLE
ScreenResolution, Start&StopTime warning

### PR DESCRIPTION
**Pull request by julia-pfarr**
- changed where to get ScreenResolution from: now getting it from "GAZE_COORDS" instead of "DISPLAY_COORDS"
- throw warning when multiple Start&StopTimes according to issue#2

### CODE REVIEW

File: eye2bids/edf2bids.py [hunk 1]
```diff
@@ -221,15 +221,12 @@ def _extract_RecordedEye(df: pd.DataFrame) -> str:
...
-                .iloc[0:1, 3:5]
-                .to_string(header=False, index=False)
-            ).split(" "),
-        )
+    return (
+        (df[df[2] == "GAZE_COORDS"])
+        .iloc[0:1, 3:5]
+        .to_string(header=False, index=False)
+        .replace(".00", "")
+        .split(" ")
```
Comment (Remi-Gau) [inline eye2bids/edf2bids.py:229]: this should return integer values and currently it is returning strings

that's why the tests are failing

File: eye2bids/edf2bids.py [hunk 2]
```diff
@@ -241,6 +238,38 @@ def _extract_TaskName(events: list[str]) -> str:
...
+    StartTime = (
+        np.array(pd.DataFrame([st.split() for st in events if st.startswith("START")])[1])
+        .astype(int)
+        .tolist()
+    )
+    if len(StartTime) > 1:
+        e2b_log.info(
+            """Your input file contains multiple start times.\n
+             As this is not seen as good practice in eyetracking experiments, only the first start time will be kept for the metadata file.\n
+             Please consider changing your code accordingly for future eyetracking experiments.\n"""
```
Comment (Remi-Gau) [inline eye2bids/edf2bids.py:251]:
```suggestion
            """Your input file contains multiple start times.\n
             As this is not seen as good practice in eyetracking experiments, \n
             only the first start time will be kept for the metadata file.\n
             Please consider changing your code accordingly for future eyetracking experiments.\n"""
```

File: eye2bids/edf2bids.py [hunk 3]
```diff
@@ -241,6 +238,38 @@ def _extract_TaskName(events: list[str]) -> str:
...
+    StopTime = (
+        np.array(pd.DataFrame([so.split() for so in events if so.startswith("END")])[1])
+        .astype(int)
+        .tolist()
+    )
+    if len(StopTime) > 1:
+        e2b_log.info(
+            """Your input file contains multiple stop times.\n
+             As this is not seen as good practice in eyetracking experiments, only the last stop time will be kept for the metadata file.\n
+             Please consider changing your code accordingly for future eyetracking experiments.\n"""
```
Comment (Remi-Gau) [inline eye2bids/edf2bids.py:267]:
```suggestion
            """Your input file contains multiple stop times.\n
             As this is not seen as good practice in eyetracking experiments, \n
             only the last stop time will be kept for the metadata file.\n
             Please consider changing your code accordingly for future eyetracking experiments.\n"""
```

File: eye2bids/edf2bids.py [hunk 4]
```diff
@@ -241,6 +238,38 @@ def _extract_TaskName(events: list[str]) -> str:
...
+
+
+def _extract_StopTime(events: list[str]) -> int:
+    StopTime = (
+        np.array(pd.DataFrame([so.split() for so in events if so.startswith("END")])[1])
+        .astype(int)
+        .tolist()
+    )
+    if len(StopTime) > 1:
+        e2b_log.info(
```
Comment (Remi-Gau) [inline eye2bids/edf2bids.py:264]:
```suggestion
        e2b_log.warning(
```

File: eye2bids/edf2bids.py [hunk 5]
```diff
@@ -241,6 +238,38 @@ def _extract_TaskName(events: list[str]) -> str:
...
 
 
+def _extract_StartTime(events: list[str]) -> int:
+    StartTime = (
+        np.array(pd.DataFrame([st.split() for st in events if st.startswith("START")])[1])
+        .astype(int)
+        .tolist()
+    )
+    if len(StartTime) > 1:
+        e2b_log.info(
```
Comment (Remi-Gau) [inline eye2bids/edf2bids.py:248]:
```suggestion
        e2b_log.warning(
```

Comment (Remi-Gau) [reply eye2bids/edf2bids.py:248]: I think we may to just use `warning.warns` instead of using the logger but we can decide on that later

**Comment by Remi-Gau** [comment]
FYI: in general it is better to tackle only one "problem" per pull request: in this one we are working on both the start / stop time AND the screen resolution. One PR for each makes it easier to review

**Review by mszinte — COMMENTED** [review]
all good

**Review by mszinte — APPROVED** [review]
instru

**Pull request merged**